docs/FAQ.md

```diff
@@ -0,0 +1,83 @@
+## Frequently Asked Questions
+
+Here are a few questions that have been asked often as well as answers from the community.
+
+If you have a question that is not answered here, feel free to ask in the Discord chat and suggest it to be placed here.
+
+Thanks!
+
+
+### Contents
+
+* [Will I make money with Zenbot?](#money)
+* [Is Docker necessary when installing Zenbot?](#docker)
+* [Why do simulations, paper trading, and live trading all yield different results?](#sim-live-differences)
+* [Why should I use simulations or paper trading if they do not reflect live trading?](#why-sim)
+* [Does Zenbot use Limit orders or Market orders?](#limit-market-orders)
+* [Why does Zenbot use Limit orders by default?](#limit-orders)
+* [Is there a community to get involved with Zenbot?](#community)
+* [Why do I keep getting a this "Nonce" error?](#error-1)
+
+
+**Will I make money with Zenbot?** <a name="money"></a>
+
+That depends… a lot. Different configurations and strategies will yield different results.
+
+The current default config and parameters will likely lose you money, so proceed with caution. Try running simulations and paper trading first to see how the bot acts (see warning below).
+
+
+
+**Is Docker necessary when installing Zenbot?** <a name="docker"></a>
+
+No, Docker is often not necessary to run Zenbot. It is often simpler to run Zenbot on a Linux machine (Ubuntu, CentOS, etc.) without Docker.
+
+If running Zenbot on Windows, Docker may be needed.
+
+
+
+**Why do simulations, paper trading, and live trading all yield different results?** <a name="sim-live-differences"></a>
+
+Simulations and paper trading almost always give overly optimistic results compared to live trading. This is because simulations and paper trading both make assumptions about when/if an order is filled.
+
+Because Zenbot defaults to using Limit orders (which often lessen fees), there tends to be much more slippage (the difference between when the bot decides to buy and when it actually buys) in live trading. Due to this, live trading is almost always worse than sims and paper trading.
+
+Also, remember that past results do not guarantee future returns.
+
+
+
+**Why should I use simulations or paper trading if they do not reflect live trading?** <a name="why-sim"></a>
+
+Simulations are more optimistic than paper trading.
+Paper trading is more optimistic than live trading.
+Therefore, if a simulation does not yield good results, odds are that neither will paper trading or (by extension) live trading.
+
+
+
+**Does Zenbot use Limit orders or Market orders?** <a name="limit-market-orders"></a>
+
+By default, Zenbot will use Limit orders, but support for Market orders has been added. 
```

Comment:
Not sure if FAQ-worthy: perhaps give a bit more info about the difference between limit order (maker) and market order (taker)?